src/utils.js

```diff
@@ -30,4 +30,35 @@ function getNodeProperty(node, path) {
   return property;
 }
 
-module.exports = getNodeProperty;
+/**
+ * Check if the file path follows an Embroider build pattern
+ * @param {string} filepath
+ * @returns boolean
+ */
+function hasEmbroiderPrefix(filepath) {
+  if (typeof filepath !== 'string') return;
+
+  const separator = filepath.includes('\\') ? '\\' : '/';
+  return filepath.split(separator).includes('embroider');
```

Comment:
Why do you have to split it to determine if the path has `embroider` in it?

Reply:
This is so I can split on `path.sep` and find "embroider" (whole word) as a file path segment.